app/controllers/admin/routes.py

```diff
@@ -85,6 +88,8 @@ def createEvent(templateid, programid=None):
             validationErrorMessage = "Unknown Error Saving Event. Please try again"
 
         if saveSuccess:
+            event = Event.get(name=eventData['name'],term=eventData['term'],timeStart=eventData['timeStart'],startDate=eventData['startDate'])
+            createLog("Created event with id:"+str(event.id)+" name: "+event.name+" startDate: "+str(event.startDate) + " found at: "+"/event/"+str(event.id)+ "/edit")
```

Comment:
Since we are showing these sentences in the log, they should be full sentences and the date should be formatted. Also, if we are going to add a link, it should be a hyperlink so the user can click on it.

Right now it looks like this 

"Created event with id:16 name: All Volunteer Training startDate: 2022-01-20 found at: /event/16/edit"

It would be better if we change it to something like: 

"Created All Volunteer Training event, with start date 01/20/2022." and maybe add a link. 

The same comment applies to other logs too.

Reply:
Also, I don't think it's necessary to include the event id.

Reply:
I would also suggest using f string instead of + to concatenate sentences. For example, instead of what's here, we could do 

`f'Created {event.name}, with start date of {event.startDate}' `